maskrcnn_benchmark/config/defaults.py

```diff
@@ -223,6 +224,64 @@
 _C.MODEL.RESNETS.RES2_OUT_CHANNELS = 256
 _C.MODEL.RESNETS.STEM_OUT_CHANNELS = 64
 
+
+# ---------------------------------------------------------------------------- #
+# RetinaNet Options (Follow the Detectron version)
+# ---------------------------------------------------------------------------- #
+_C.MODEL.RETINANET = CN()
+
+# This is the number of foreground classes, background is not included.
```

Comment:
Yes, the comment is wrong.  For the second question, definitely, one place is much better. But I would like it is under MODEL instead of MODEL.ROI_BOX_HEAD.NUM_CLASSES.

Reply:
Oh yes, `MODEL.ROI_BOX_HEAD.NUM_CLASSES` is not a good place to take RetinaNet into account. Let me think about it.

Also, I haven't looked closely at the retinanet implementation of Detectron, but don't we have a `background` class in the classifier?

Reply:
I'd rather keep the `ROI_BOX_HEAD.NUM_CLASSES` for now, as when it was defined it was a property of `ROI_HEAD`, and different heads can have different number of classes.

Thanks for the explanation on the second question!

Reply:
I think that makes sense.